README.md

```diff
@@ -1,17 +1,38 @@
-# Sample
+﻿# ABOUT
 
-This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
+Sample.
 
-### Credits for images:
+This module is for fun and trying out new crazy and wacky things with the game. 
+It houses a number of example blocks and UI screens.
 
-WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+ 
+A person can test their UI screens and other various creations in this module.
+This module is open to local developments and tests.
+It is more inclined towards developers who want to test stuff; but players can enjoy it too!
+
+# LICENSE
+
+The module is licensed under Apache 2.0 license.
+
+# CREDITS
+
+Credits for images:
 
-CementStone: Dane Claessen (https://github.com/Dane99)
+WhiteStone: http://www.planetminecraft.com/texture_pack/new-start---by-loufisch/
+Credits for CementStone: Dane Claessen (https://github.com/Dane99)
 
+Credits for 
 Rainbow block: Dev Choganwala (https://github.com/DevChoganwala)
 
-Diamond wallpaper: [Vishal Devireddy](https://github.com/TwoTau)
 
-Giftbox: [András Ottó Földes](https://github.com/andriii25)
+Credits for Diamond wallpaper: [Vishal Devireddy](https://github.com/TwoTau)
+
+
+Credits for Giftbox: [András Ottó Földes](https://github.com/andriii25)
+
+Credits for 
+Blue Deco: Megan Yu (https://github.com/meganyyu)
+
+# LINK
```

Comment:
We don't need this section - this readme is primarily viewed when the user visits the Sample repo. Remove it.